akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -474,6 +472,80 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>standalone</id>
+            <properties>
+                <docker.base.image>eclipse-temurin:21.0.1_12-jre-jammy</docker.base.image>
+            </properties>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>io.kalix</groupId>
+                        <artifactId>akka-runtime-core_2.13</artifactId>
+                        <version>${kalix-runtime.version}</version>
+                    </dependency>
+                </dependencies>
+            </dependencyManagement>
+
+            <dependencies>
+                <dependency>
+                    <groupId>io.kalix</groupId>
+                    <artifactId>akka-runtime-core_2.13</artifactId>
+                </dependency>
+            </dependencies>
+
+            <build>
+                <pluginManagement>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>${docker-maven-plugin.version}</version>
+                        <configuration combine.self="override">
```

Comment:
`combine.self="override"` is important, otherwise it will merge with the other plugin definition